### ISSUE TITLE
Some small islands like Aruba are not displayed on map

### ISSUE THREAD

**Issue by pascalheraud** [tmrowco/electricitymap#1153]
The simplification process contained into generate-geometries.js simplifies the geometries of the entire world by using a weight of 0.01.
This threshold is so high that some small islands like Aruba (and others from Caribbean) are simply removed from map.

With this threshold of 0.01, the generated file world.json weights 420kb.

With a threshold of 0.002, Aruba come back into map but the  world.json grows up to 896kb. E-Map still display well on my computer.

The good deal would be to :
- apply a threshold of 0.01 to all countries but Aruba.
- apply a threshold of 0.002 for Aruba
- merge the two previous in world.json

I tried to figure out but didn't succeed using topojson. My guess was that topojson.mergeArcs would be a could candidate for merging two topojson.

Please comment on this issue how to merge the two topojson or any other idea.

**Comment by systemcatch** [comment]
Looking at the topojson simplify documentation https://github.com/topojson/topojson-simplify I think you have the right idea here. We need to apply a different threshold to Aruba while preserving the rest of the map. @maxbellec any thoughts?

**Comment by maxbellec** [comment]
Same as @systemcatch, I completely agree with the general idea. Using `topojson` is not always straightforward... If you don't already know it, [geojson.io](http://geojson.io/) is a great tool, it also recognises `topojson`.

**Comment by pascalheraud** [comment]
Looking deeper into "merge" from topojson shows me that it's designed to merge some polygon into one topology.

I wrote a function to merge two different topologies so we can apply different simplify thresholds, for example : 0.01 for entire world and 0.001 for Aruba for instance.

After simplifying separately, we can merge both topologies and generate world.json containing Aruba.

I just added an attribute "lessSimplify" to zones into zoneDefinitions. If it's present the zone will be simplified to 0.001 instead of 0.01.

This approach seems working. I just need to clean up my code and I'll send a PR soon.
